spec/requests/posts_request_spec.rb

```diff
@@ -146,4 +146,59 @@
       end
     end
   end
+
+  describe '#destroy_post_request' do
+    let(:new_post) { create :post, user: }
+    image_path = File.join(File.dirname(__FILE__), '..', 'images', 'download.jpeg')
+    let(:user) { create :user }
+    let(:commentable) { create :comment, user:, commentable: new_post }
+    let(:comment) { create :comment, user:, commentable: }
```

Comment:
Let's re order for readability
```suggestion
    image_path = File.join(File.dirname(__FILE__), '..', 'images', 'download.jpeg')
    let(:user) { create :user }
    let(:new_post) { create :post, user: }
    let(:commentable) { create :comment, user:, commentable: new_post }
    let(:comment) { create :comment, user:, commentable: }
```